diff -Nurp gearbox_orig/src/flexiport/logfile.h gearbox_ros/src/flexiport/logfile.h
--- gearbox_orig/src/flexiport/logfile.h	2013-07-26 19:34:27.029131611 -0700
+++ gearbox_ros/src/flexiport/logfile.h	2013-07-26 19:39:09.234176829 -0700
@@ -38,6 +38,7 @@
 
 #include "timeout.h"
 #include "flexiport_types.h"
+#include "unistd.h"
 
 namespace flexiport
 {
diff -Nurp gearbox_orig/src/flexiport/port.h gearbox_ros/src/flexiport/port.h
--- gearbox_orig/src/flexiport/port.h	2013-07-26 19:34:27.029131611 -0700
+++ gearbox_ros/src/flexiport/port.h	2013-07-26 19:40:06.951023535 -0700
@@ -45,6 +45,7 @@
 
 #include "flexiport_types.h"
 #include "timeout.h"
+#include "unistd.h"
 
 /** @ingroup gbx_library_flexiport
 @{
diff -Nurp gearbox_orig/src/hokuyo_aist/python/CMakeLists.txt gearbox_ros/src/hokuyo_aist/python/CMakeLists.txt
--- gearbox_orig/src/hokuyo_aist/python/CMakeLists.txt	2013-07-26 19:34:28.137685660 -0700
+++ gearbox_ros/src/hokuyo_aist/python/CMakeLists.txt	2013-07-26 19:38:36.625879302 -0700
@@ -41,7 +41,7 @@ else(CMAKE_MAJOR_VERSION EQUAL 2 AND CMA
 		endif(Boost_USE_MULTITHREAD)
 		MARK_AS_ADVANCED (BOOST_LIB_SUFFIX)
 
-		SET (boostPythonLib boost_python${BOOST_LIB_SUFFIX})
+		SET (boostPythonLib boost_python)
 		INCLUDE_DIRECTORIES (${Boost_INCLUDE_DIRS})
 		LINK_DIRECTORIES (${Boost_LIBRARY_DIRS})
 	else(Boost_FOUND)
